<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpeningFolderClass</class>
 <widget class="QDialog" name="DialogOpeningFolderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="bnOk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>376</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bnCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>376</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>497</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Opening folder...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>331</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurFile">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>306</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Currently opening</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bnOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogOpeningFolderClass</receiver>
   <slot>OnBnOKclicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogOpeningFolderClass</receiver>
   <slot>OnBnCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnBnOKclicked()</slot>
  <slot>OnBnCancelClicked()</slot>
 </slots>
</ui>
